services/user/fractally-world.wit

```diff
@@ -1,84 +1,83 @@
-interface fractally-types {
-    record config {
-        // @velua, you spent plenty of time on this for the Psinq UI. Would you fill in the minimal set of details we determined were necessary for meeting schedule?
+//// SERVICE
+
+interface types {
+
+    record eval {
+        id: u32, // primary key
+        type: u8, // 0 = reputation, 1 = favor, etc.
     }
-    record fractal-info {
-        // What details do we need here as a "profile" for a fractal?
-        name: string;
-        bio: string;
-        avatar: string;
-        created-at: timestamp;
-        status: enum<pre-evaluations, active, inactive>;
-        num-members: u64;
+
+    record member {
+        account: string, // primary key
+        name: string,
+        joined-at: timestamp,
+        reputation: u32,
+        candidate: bool,
     }
 }
 
-interface fractally {
-    use fractally-types.{config};
-    set-config: func(config: config) -> result<void, error>;
-    get-fractal-info: func() -> result<fractal-info, error>;
-    // I propose the periodic subj evaluation process is managed via FSM with this method to nudge it when needed (rather than numerous such do-the-next-thing methods)
-    nudge: func() -> result<void, error>;
-}
+interface fractal {
+    use types.{config};
```

Comment:
you don't define `config` above. Is there any config? Not sure after all the deletion that there's anything for fractal config

Reply:
no longer applicable